### PR TITLE
MAT-6682: Update email wording

### PR DESCRIPTION
**Pull request by mcmcphillips**
## PR Template

Jira Ticket: [MAT-6682](https://jira.cms.gov/browse/MAT-6682)
(Optional) Related Tickets:

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included
* [ ] No extemporaneous files are included (i.e. Complied files or testing results)
* [ ] This PR is into the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e. Changing auth methods, Adding a new user type, Adding a required but vulnerable package)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: lambda-function/src/app.ts [hunk 1]
```diff
@@ -38,43 +38,57 @@ export const lambdaHandler = async (event: S3Event): Promise<Measure> => {
...
       madieMeasure,
       matMeasure.harpId,
     );
-    logAndMail(`Transferred Measure id: ${response.id}`);
-    emailMessage = `${emailMessage}\nYour measure transfer was successful.`;
-    console.log("Lambda execution completed...");
-    await mailService.sendMail(emailId, "Successfully imported the Measure", emailMessage);
+    if (response) {
+      /* Success email */
+      logAndMail("--------Measure Details-------");
```
Comment (adongare) [inline lambda-function/src/app.ts:57]: just thinking, would it be good if we introduce some templates like a handlebar or is it too much for this use case?

Comment (mcmcphillips) [reply lambda-function/src/app.ts:57]: I would think it might be a little too much for three possible hardcoded cases

**Review by adongare — APPROVED** [review]
